client/lib/analytics/test/gpc-opt-out.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * @jest-environment jsdom
+ */
+import { mayWeTrackUserGpcInCCPARegion, isGpcFlagSetOptOut } from '../utils';
+import isRegionInCcpaZone from '../utils/is-region-in-ccpa-zone';
+
+// Return a predictable value for whether the user is in a CCPA region.
+jest.mock( '../utils/is-region-in-ccpa-zone' );
+
+let windowSpy = jest.SpyInstance;
+
+// Return a predictable value for window.navigator.globalPrivacyControl.
+const setMockGPCValue = ( gpcValue ) => {
+	windowSpy.mockImplementation( () => ( {
+		navigator: {
+			globalPrivacyControl: gpcValue,
+		},
+	} ) );
+};
+
+describe( 'global privacy control', () => {
+	beforeEach( () => {
+		windowSpy = jest.spyOn( global, 'window', 'get' );
+	} );
+
+	afterEach( () => {
+		windowSpy.mockRestore();
+	} );
+
+	describe( 'mayWeTrackUserGpcInCCPARegion', () => {
```

Comment:
If that has to be unit tests, we should mock the `isGpcFlagSetOptOut` function when testing the `mayWeTrackUserGpcInCCPARegion`. Especially if we test that function below.

Reply:
I agree. I removed the tests for the browser flag.